frontend/src/views/Home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from '../components/Button/Button';
+
+const Home = () => (
+  <div className="Home">
+    <div className="Home-hero">
+      <h1 className="Home-heading">Two line description<br />of the idea and service.</h1>
+      <p className="Home-text">Just a tiny bit more explanation, since two lines probably won’t tell enough.</p>
```

Comment:
I thought of this style as being represented by `.intro` maybe you can check out if it works. You'll probably still need the extra margin-top but then we can re-use `.intro` across components.

Reply:
Again, 100% right!